app/src/main/java/com/example/tawazz/download/ExtensionType.java

```diff
@@ -0,0 +1,13 @@
+package com.example.tawazz.download;
+
+import androidx.annotation.NonNull;
+
+public enum ExtensionType {
+    JPG {
+        @NonNull
+        @Override
+        public String toString() {
```

Comment:
i dont like overriding toString this way especially in an enum; maybe add abstract method "getExtensionText" (or some better name)?